xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -1080,20 +1080,21 @@ bool CGUIWindowVideoBase::OnPlayMedia(int iItem, const std::string &player)
   CServiceBroker::GetPlaylistPlayer().Reset();
   CServiceBroker::GetPlaylistPlayer().SetCurrentPlaylist(PLAYLIST::TYPE_NONE);
 
-  CFileItem item(*pItem);
+  const CFileItemPtr itemCopy = std::make_shared<CFileItem>(*pItem);
```

Comment:
Iirc there was a desire to kill cfileitemptr. Can you please use auto here since it is implicit from the constructor?
In other places you'd write the actual type instead of the typedef but here it's one of those places where auto would suffice.